tests/test_vector2_truncate.py

```diff
@@ -19,6 +19,9 @@ def test_truncate_invariant(x: Vector2, max_length: float):
 
 
 @given(x=vectors(max_magnitude=1e75), max_length=floats(min_value=0, max_value=1e75))
+@example( # Large example where x.length == max_length but 1 * x != x
+    x=Vector2(0.0, 7.387424005855793e+62), max_length=7.387424005855793e+62
```

Comment:
Can you manually simplify this? Or is this the simplest form?

Reply:
I don't know if it's the simplest form, but I was unable to reproduce the issue with `7e+62` or `8e+62`

Reply:
Nevermind, the issue can be reproduced with `7.1e+62`.